src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Figures implements Forms {
```

Comment:
Название классов принято писать в единичном числе. Давай сделаем этот класс абстрактным, так как мы не можем создать просто какую-то фигуру
```suggestion
public class Figure implements Forms {
```